appcelerator-archive/amp
# Faster CLI test

This PR reduces CLI tests time.

## How to test
```
$ ampmake clean build
$ amp cluster create
$ TESTINCLUDE=tests/cli/testing testrunner tests/cli
```
Check Travis build.
